### PR TITLE
fix(frontend): Quickfix sorting for Token Groups

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

It may happen that after the grouping, a particular group results with a balance that is greater than another group/token above it. So the sorting could be wrong:

image

We are going to refactor the grouping util, but not in this PR.

For now, it was requested a "quickfix" to at least have the correct sorting. To achieve that, we sort the final list of `[tokens + groups]` by balances (token and USD).

It is guaranteed that the only sorting criteria that is being mixed is the one concerning the balances, because a group could only have a greater (or same) balance than the single tokens it was born from.

The other sorting criteria (name, network.name) can be ignored because the resulting position given by them will be maintained during grouping.

### CODE REVIEW

**Comment by AntonioVentilii-DFINITY** [comment]
[OISY-244]

File: src/frontend/src/lib/utils/token-group.utils.ts [hunk 1]
```diff
@@ -81,7 +82,11 @@ export const groupTokensByTwin = (tokens: TokenUi[]): TokenUiOrGroupUi[] => {
 		return token;
 	});
 
-	return mappedTokensWithGroups.filter(
-		(t) => isTokenUiGroup(t) || !groupedTokenTwins.has(t.symbol)
-	);
+	return mappedTokensWithGroups
+		.filter((t) => isTokenUiGroup(t) || !groupedTokenTwins.has(t.symbol))
+		.sort(
```
Comment (peterpeterparker) [inline src/frontend/src/lib/utils/token-group.utils.ts:87]: Can we have a test?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/utils/token-group.utils.ts:87]: Sorry, forgot! Just included, have a look please

**Review by peterpeterparker — APPROVED** [review]
LGTm, thx

**Pull request merged**